ocs_ci/ocs/machine.py

```diff
@@ -275,6 +298,39 @@ def create_custom_machineset(
         raise UnsupportedPlatformError("Functionality not supported in UPI")
 
 
+def create_infra_nodes(num_nodes):
```

Comment:
My understanding when we say infra we not only label the nodes as infra, we also taint the infra nodes to enable OCS services on nodes. so I feel the name of the function can be create_ocs_infra_nodes().

Reply:
sure